### PR TITLE
Development

### PR DESCRIPTION
**Pull request by pankajan05**

### CODE REVIEW

File: js/components.js [hunk 1]
```diff
@@ -151,31 +151,63 @@ document.addEventListener("DOMContentLoaded", function () {
...
+  const homeAnchor = document.getElementById('home-anchor');
   const navLinks = document.querySelectorAll("#navbar-placeholder a");
+  
 
   navLinks.forEach((link) => {
-    if (link.getAttribute("href").endsWith(currentPage)) {
-      link.classList.add("active");
-      // Also set parent link active if a dropdown menu is active
+
+    switch (link.getAttribute("href").endsWith(currentPage) && link.getAttribute("href").endsWith(currentPageHome)) {
```
Comment (pankajan05) [inline js/components.js:167]: Wrong use of Switch

File: js/components.js [hunk 2]
```diff
@@ -151,31 +151,63 @@ document.addEventListener("DOMContentLoaded", function () {
 });
 
 function setActiveLink() {
+  console.log(
```
[the remark this answers is not in the document]
Comment (DanushB151) [reply js/components.js]: removed

File: js/components.js (hunk 1 above)
Comment (DanushB151) [reply js/components.js]: solved

**Review by pankajan05 — COMMENTED** [review]
checked

**Pull request merged**